Userland/Libraries/LibJS/Runtime/GlobalObject.h

```diff
@@ -34,8 +34,8 @@ class GlobalObject : public ScopeObject {
     Shape* new_object_shape() { return m_new_object_shape; }
     Shape* new_script_function_prototype_object_shape() { return m_new_script_function_prototype_object_shape; }
 
-    // Not included in JS_ENUMERATE_NATIVE_OBJECTS due to missing distinct prototype
```

Comment:
I'd rather keep the comment here, it applies to both now

Reply:
I only removed it because of the comments on the actual members. Should I keep them in both places?

Reply:
Let's do that, yeah. Makes it clear immediately without having to scroll around :)